Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java

```diff
@@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mReactInstanceManager = ReactInstanceManager.builder()
         .setApplication(getApplication())
-        .setBundleAssetName("MoviesApp.android.bundle")
+        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
```

Comment:
@astreet How should we handle the use case of "reloading" the
ReactInstance? One of the nice things about the iOS API is that the
delegate can change the `sourceURL` method and call `reload` to refresh the
React instance:
https://github.com/facebook/react-native/blob/master/React/Base/RCTBridge.m#L237

One approach would be to allow the "bundleFile" of a `ReactInstance` to be
mutable. Would this be preferred over this interface?

On Tue, Oct 20, 2015 at 1:46 PM, Andy Street <notifications@github.com>
wrote:

> In
> Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java
> <https://github.com/facebook/react-native/pull/3189#discussion_r42528691>:
>
> > @@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
> >
> >      mReactInstanceManager = ReactInstanceManager.builder()
> >          .setApplication(getApplication())
> > -        .setBundleAssetName("MoviesApp.android.bundle")
> > +        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
>
> Were there objections to just going with
> setBundleFile(file://android_assets/MoviesApp.android.bundle)? (or
> assets:// if that doesn't work).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/facebook/react-native/pull/3189/files#r42528691>.
>